addons/point_of_sale/static/src/app/screens/partner_list/partner_editor/partner_editor.js

```diff
@@ -108,8 +108,17 @@ export class PartnerEditor extends Component {
             });
         }
         processedChanges.id = this.props.partner.id || false;
-        await this.orm.call("res.partner", "create_from_ui", [processedChanges]);
```

Comment:
No need for the `create_from_ui` from the res.partner model? Maybe we can also remove it.